setup.sh

```diff
@@ -67,14 +67,13 @@ function setupSwap() {
 
 function installSSH() {
     echo "--------------------------------------------"
-    echo "Installing SSH..."
-    sudo apt-get install openssh-client
+    # echo "Installing SSH..."
+    # sudo apt-get install openssh-client
     read -p "Generating SSH keys. Enter email address to be associated: " email
```

Comment:
True. I set this up with the user setting up SSH keys to add to GitHub in mind. In fact, just ssh-keygen would do the trick.